addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -38,27 +41,121 @@ The following parameters are automatically added:
 All Things support the following channels:
 
 | Channel Type ID | Item Type    | Description  |
-|-----------------|------------------------|--------------|----------------- |------------- |
+|-----------------|------------------------|--------------|
 | input | String       | Input parameter to provide to the command |
 | output | String       | Output of the last execution of the command |
 | exit | Number       | The exit value of the last execution of the command |
 | run | Switch       | Send ON to execute the command and the current state tells whether it is running or not |
 | lastexecution | DateTime       | Time/Date the command was last executed, in yyyy-MM-dd'T'HH:mm:ss.SSSZ format |
 
-## Full Example
+## Short Example
 
 **demo.things**
 
-```
-Thing exec:command:apc [command="/usr/local/bin/apcaccess  status", interval=15, timeout=5]
+```java
+Thing exec:command:apc [command="/usr/local/bin/apcaccess status", interval=15, timeout=5]
 Thing exec:command:myscript [command="php ./configurations/scripts/script.php %2$s", transform="REGEX((.*?))"]
 ```
 
 **demo.items**
 
-```
-String APCRaw "[%s]" (All) {channel="exec:command:apc:output"} 
-Switch APCRunning { channel="exec:command:apc:run"}
+```java
+String APCRaw "[%s]" (All) {channel="exec:command:apc:output"}
+Switch APCRunning {channel="exec:command:apc:run"}
 Number APCExitValue {channel="exec:command:apc:exit"}
 DateTime APCLastExecution {channel="exec:command:apc:lastexecution"}
 ```
+
+## Full Example
+
+Following is a example shown how to read out the temperature of the RPI with all files needed to set up and debug or log the executed command.
+The folder structure is for an installation with apt-get, for a different installation setup look [here](http://docs.openhab.org/installation/linux.html#file-locations) or search for them in your drive.
+
+First we need to check if the user openhab is able to execute the command we want to execute.
+The first command is executet as the user we are logged in the second as user openhab.
+If you get an error or an massages which indicates the command does not execute properly as user openhab then try to google the needed permissions and set them for the user openhab.
+More details [here](https://community.openhab.org/t/2-openhab2-rpi-system-temperature-chart-with-persistence/35182) and [here](https://community.openhab.org/t/openhab-sudo-exec-binding/34988).
+
+```shell
+cat /sys/class/thermal/thermal_zone0/temp
+48312
+
+sudo -u openhab cat /sys/class/thermal/thermal_zone0/temp
+47774
+```
+
+As the command got executed as user `openhab` we can proceed to set up our OpenHAB.
+
+First we need a **Thing** file which configures the command line call to execute.
+
+```shell
+sudo nano /etc/openhab2/things/exec.things
+```
```

Comment:
Please leave these out. Instead modify the text above to read something like:

> First we need to create or modify a Thing file which configures the command line call to execute:

Rationale: Where a thing file is located is stated in another part of the documentation. How the user edits the file (here using nano) is not ours to decide.